e2e/README.md

```diff
@@ -5,18 +5,28 @@ In order to run it locally you can do the following
 eval $(minikube -p minikube docker-env)
 ```
 
+Note that the control plane flags in `e2e-values.yml` have been set meeting the minikube specifications. 
+
 Then you need to build the binary and the image. Notice that  since the Dockerfile includes multiarch
 support, you may need to set `DOCKER_BUILDKIT=1` when running `docker build` for the `TARGETARCH`
 and `TARGETOS` args to be populated.
 ```shell
-GOOS=linux GOARCH=amd64 make compile
-docker build -t test_image_normal:test  .
+GOOS=linux GOARCH=amd64 make compile # Set GOOS and GOARCH explicitly since Dockerfile expects them in the binary name
+export  DOCKER_BUILDKIT=1
+docker build -t test_image:test --build-arg 'MODE=normal' .
+docker build -t test_image:test-unprivileged --build-arg 'MODE=unprivileged' .
 ```
 
-Then you can run manually any scenario you are interested into
+You need to install the binary `https://github.com/newrelic/newrelic-integration-e2e-action/tree/main/newrelic-integration-e2e` used in the e2e test
 ```shell
-helm dependencies update ./e2e/charts/newrelic-infrastructure-k8s-e2e
-go run e2e/cmd/e2e.go --verbose --cluster_name=e2e --nr_license_key="fakeLicense" --rbac=true --integration_image_tag=test --integration_image_repository=test_image_normal
+git clone https://github.com/newrelic/newrelic-integration-e2e-action
+cd newrelic-integration-e2e
+go build -o  $GOPATH/bin/newrelic-integration-e2e ./...
 ```
 
-Note: On macOS you might have to do run `minikube start --vm` to get a proper IP that will be used by the e2e test to check the cluster flavor.
+You can now run the e2e tests locally
+```shell
+LICENSE_KEY=${LICENSE_KEY} newrelic-integration-e2e --commit_sha=test-string --retry_attempts=5 --retry_seconds=60 \
```

Comment:
Yes, but the command is quite long:
```
$GOPATH/src/github.com/newrelic/newrelic-integration-e2e-action/newrelic-integration-e2e/cmd/main.go
```

I would prefer to install it just once😕

Reply:
Moreover the fact that the go.mod is not in the root folder creates a lot of issues when running the binary

Reply:
What does `LICENSE_KEY=${LICENSE_KEY}` accomplish at the beginning of the line?

Reply:
the action is currently expecting it both as a env var and as an argument. That is an issue merely running it locally, in the ci/cd we did not noticed it since the env var was already there https://github.com/newrelic/newrelic-integration-e2e-action/issues/8